android/src/main/java/com/thebluealliance/androidclient/Utilities.java

```diff
@@ -1,31 +1,36 @@
 package com.thebluealliance.androidclient;
 
+import com.google.common.base.Predicate;
+
+import com.thebluealliance.androidclient.activities.GamedayActivity;
```

Comment:
Yes, that code-style file performed these import changes. I added a note in the README about where to install that file on OSX.